video/out/vulkan/context_display.c

```diff
@@ -278,14 +285,84 @@ struct priv {
     struct vulkan_display_opts *opts;
     uint32_t width;
     uint32_t height;
+
+#if HAVE_DRM
+    struct mpv_opengl_drm_params_v2 drm_params;
+#endif
 };
 
+#if HAVE_DRM
+static bool open_render_fd(struct ra_ctx *ctx, const char *render_path)
+{
+    struct priv *p = ctx->priv;
+    p->drm_params.fd = -1;
+    p->drm_params.render_fd = -1;
+
+    int fd = open(render_path, O_RDWR | O_CLOEXEC);
+    p->drm_params.render_fd = fd;
+    return fd != -1;
```

Comment:
Bunch of unnecessary code here - kill the temporary variable and useless initialisation.

```
    struct priv *p = ctx->priv;
    p->drm_params.fd = -1;
    p->drm_params.render_fd = open(render_path, O_RDWR | O_CLOEXEC);

    return p->drm_params.render_fd != -1;
```